docs: update emacs lsp-mode instructions (#60)
elixir-tools/credo-language-server: README.md

<details>
<summary>Emacs</summary>

#### Using lsp-mode:

> Credo is part of lsp-mode and the language server can be installed by
> running `M-x lsp-install-server credo-language-server`.

Reviewer: ```suggestion
credo-language-server is included with [lsp-mode](https://github.com/emacs-lsp/lsp-mode) and can be installed by running `M-x lsp-install-server credo-language-server`.
```

You might want to set the lsp-credo-version to the latest release:

> ```

Reviewer: ```suggestion
```elisp
```
